modules/users/client/views/profile/profile-view-about.client.view.html

```diff
@@ -10,15 +10,14 @@
       <div class="panel-body">
 
         <!-- Short descriptions -->
-        <div ng-if="profileCtrl.profile.description.length < 2400" ng-bind-html="profileCtrl.profile.description | trustedHtml"></div>
+        <div ng-if="profileCtrl.profile.description.length < 2400 || profileCtrl.profileDescriptionToggle"
+        ng-bind-html="profileCtrl.profile.description | trustedHtml"></div>
```

Comment:
Nit; `ng-bind-html` should have a few more spaces to match the position of `ng-if`